composer.json

```diff
@@ -24,7 +24,7 @@
     "require-dev": {
         "inertiajs/inertia-laravel": "^0.6.5",
         "laravel/sanctum": "^3.0",
-        "livewire/livewire": "^2.12",
+        "livewire/livewire": "dev-main",
```

Comment:
dev-main isn't the safest to pin to - maybe a specific beta version or a wild-card beta version lock